docs/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Changelog
 - Enh #48: Enhanced menu item visibility configuration
 - Enh #40: Emoji in headline creates new paragraph
 - Chng: Refactored history into own plugin
+- Enh #1: Update JS dependencies
```

Comment:
Can you please use the # id of this PR here.

I've described the Changelog Scheme here: 

https://community.humhub.com/s/contribution-core-development/wiki/Code+Style#pull-requests-%26-changelog

Reply:
Changed